nservicebus/upgrades/asp-6to1.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Azure Storage Persistence Upgrade Version 6 to 1
```

Comment:
Struggled with this one too. Like you I couldn't come up with anything better that didn't turn into a 30 word sentence.